tests/Doctrine/DBAL/Migrations/Tests/Finder/RecursiveRegexFinderTest.php

```diff
@@ -63,6 +63,7 @@ public function testFindMigrationsReturnsTheExpectedFilesFromDirectory()
             '20150502000004' => 'TestMigrations\\Version20150502000004',
             '20150502000005' => 'TestMigrations\\Version20150502000005',
             '1_reset_versions' => 'TestMigrations\\Version1_reset_versions',
+            '1_symlinked_file' => 'TestMigrations\\Version1_symlinked_file',
```

Comment:
Yes, it is pretty much the same but you're testing two different things:

1. the behavior without any symlinks
2. the behavior with symlinks

That difference is enough to deserve another test with a name that reflects what you're testing. That's my opinion, anyway. @mikeSimonson is the maintainer here, so it's ultimately his call.

To make it simpler you could make a new directory and symlink the `_files` directory into it and make sure the symlink is followed.